packages/app-root/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@zooniverse/app-root",
+  "description": "",
+  "license": "Apache-2.0",
+  "author": "Zooniverse <contact@zooniverse.org> (https://www.zooniverse.org/)",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@zooniverse/async-states": "~0.0.1",
+    "@zooniverse/grommet-theme": "~3.1.1",
+    "@zooniverse/panoptes-js": "~0.4.1",
+    "@zooniverse/react-components": "~1.6.1",
+    "grommet": "~2.33.2",
+    "grommet-icons": "~4.11.0",
+    "next": "13.5.2",
```

Comment:
We should probably keep this app and the content pages app on the same version of `next`. Content pages is on 13.4 but I’ve opened #5411 to upgrade to 13.5.

Reply:
True! I've assigned myself #5411 to review too